### DISCUSSION TITLE
load tablecache  Very slow

### DISCUSSION THREAD

**Discussion by fin-illusion** [dbeaver/dbeaver#15015] [Q&A]
oracle load tablecache  Very slow

**Comment by LonwoLonwo** [comment]
Hello @fin-illusion 

Please do not ignore the issue template.

DBeaver has some extra performance options. You can find them in your "Connection Settings" -> "Oracle properties".

![2021-12-22 15_14_38-Connection _orcl_ configuration](https://user-images.githubusercontent.com/45152336/147091465-2e5c1c27-462d-4be2-bbbd-2a7ec2a9d851.png)

**Comment by fin-illusion** [comment]
Thank you. This setting is all right.
